03-Bash/find_dup_files/find_dup_files.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+
+
+current_dir="$(pwd)"
+
+function view_help {
+	echo "Script find_dup_files.sh usage information."
+	echo "SYNOPSIS"
+	echo "find_dup_files.sh [DIR] [-h]"
+	echo -e "Display information about files with identical md5, if the directory is not specified, then the current directory is by default.\n"
```

Comment:
Too long line

Reply:
Ok. Thank you. I am changing it.

Reply:
I don't see any change in this file. Please check the updated branch.